### PR TITLE
open-cluster-management/backlog#4737 ts secret

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
@TheRealHaoLiu : Here is a draft of the troubleshooting topic for 4731, but I just saw that this is just closed. Is this not necessary now? Thanks!

**Comment by cadawson** [comment]
Hi Han. Hao suggested that I ask you to take a look at this PR. Thanks!

File: troubleshooting/trouble_import_fail_secret.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+----
+Failed to fetch import yaml secret
+----
+
+[#identify-import-fail-secret]
+== Identifying the problem: Troubleshooting a failed imported cluster secret after upgrade
+
+To confirm that the issue can be resolved by the steps that follow, complete the following steps:
+. Run the following command to change to your project namespace:
```
Comment (hanqiuzh) [inline troubleshooting/trouble_import_fail_secret.adoc]: feel `project namespace` is obscure, here is saying the ACM namespace. shall we say something like `ACM install namespace (default open-cluster-management)`?

File: troubleshooting/trouble_import_fail_secret.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
++
+----
+oc project <namespace>
+----
++
+Replace _<namespace>_ with your project namespace.
+
+. Run the following command to determine if the `managedcluster-import-controller` is running:
```
Comment (hanqiuzh) [inline troubleshooting/trouble_import_fail_secret.adoc]: the command is not checking if `managedcluster-import-controller` is running. 

The command is checking if `managedcluster-import-controller` has the required permission.

File: troubleshooting/trouble_import_fail_secret.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+[#resolving-import-fail-secret]
+== Resolving the problem: Troubleshooting a failed imported cluster secret after upgrade
+
+To resolve the problem, enter the following command to restart the `multicluster-operators-standalone-subscription` service:
++
+----
+oc delete $(oc get pod -o name | grep multicluster-operators-standalone-subscription)
+----
+Replace _name_ with the name of the pod that you were trying to import.
```
Comment (hanqiuzh) [inline troubleshooting/trouble_import_fail_secret.adoc]: this  line "Replace _name_ with the name of the pod that you were trying to import."  is not needed as the command can work without any name replaces

**Review by hanqiuzh — CHANGES_REQUESTED** [review]

File: troubleshooting/trouble_import_fail_secret.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[#trouble-import-fail-secret]
+= Troubleshooting a failed imported cluster secret after upgrade 
+
+[#symptom-import-fail-secret]
+== Symptom: Troubleshooting a failed imported cluster secret after upgrade
+
+After an upgrade, your cluster import in the {product-title} console might fail with the following message:
```
Comment (hanqiuzh) [inline troubleshooting/trouble_import_fail_secret.adoc]: shall we specify it's from 2.0.0 to 2.0.1, this issue only applies for 2.0.0->2.0.1 upgrade

**Comment by hanqiuzh** [comment]
@cadawson  hi, left some comment 👍

**Review by hanqiuzh — APPROVED** [review]

File: troubleshooting/trouble_import_fail_secret.adoc (hunk 1 above)
Comment (cadawson) [reply troubleshooting/trouble_import_fail_secret.adoc]: Clarified

File: troubleshooting/trouble_import_fail_secret.adoc (hunk 2 above)
Comment (cadawson) [reply troubleshooting/trouble_import_fail_secret.adoc]: Fixed

File: troubleshooting/trouble_import_fail_secret.adoc (hunk 3 above)
Comment (cadawson) [reply troubleshooting/trouble_import_fail_secret.adoc]: Removed

File: troubleshooting/trouble_import_fail_secret.adoc (hunk 4 above)
Comment (hanqiuzh) [reply troubleshooting/trouble_import_fail_secret.adoc]: I think it's ok to keep it as it. For upgrade it has to be >2.0.0 😂

**Comment by hanqiuzh** [comment]
Looks good to me

**Review by hanqiuzh — APPROVED** [review]

File: troubleshooting/trouble_import_fail_secret.adoc (hunk 4 above)
Comment (cadawson) [reply troubleshooting/trouble_import_fail_secret.adoc]: Added versions

**Pull request merged**